torchci/lib/drciUtils.ts

```diff
@@ -19,6 +19,8 @@ This comment was automatically generated by Dr. CI and updates every 15 minutes.
 export const HUD_URL = "https://hud.pytorch.org/pr/";
 export const BOT_COMMANDS_WIKI_URL =
   "https://github.com/pytorch/pytorch/wiki/Bot-commands";
+export const FLAKY_RULES_JSON =
+  "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/rules.json";
```

Comment:
Could we use a more descriptive name for this one because `rules.json` could mean anything? Names like `flaky-rules.json` or `ignoreable-rules.json` are both fine IMO as along as the constant here is consistent with it

Reply:
changed to flaky-rules.json